Exam-Service/src/main/java/tw/waterball/judgegirl/examservice/usecases/CreateQuestionUseCase.java

```diff
@@ -0,0 +1,34 @@
+package tw.waterball.judgegirl.examservice.usecases;
+
+import lombok.Data;
+import tw.waterball.judgegirl.entities.Question;
+import tw.waterball.judgegirl.examservice.repositories.QuestionRepository;
+
+import javax.inject.Named;
+
+@Named
+public class CreateQuestionUseCase {
+    private final QuestionRepository questionRepository;
+
+    public CreateQuestionUseCase(QuestionRepository questionRepository) {
+        this.questionRepository = questionRepository;
+    }
+
+    public void execute(Request request, Presenter presenter) {
+        Question question = new Question(request.examId, request.problemId, request.quota, request.score);
+        presenter.setQuestion(questionRepository.save(question));
+    }
+
+    public interface Presenter {
+        void setQuestion(Question question);
+    }
+
+    @Data
+    public static class Request {
+        public Integer examId = null;
+        public Integer problemId = null;
+        public Integer quota = null;
+        public Integer score = null;
+    }

```

Comment:
use `int` instead of `Integer`

**only use Integer when the value of it can be null**